test/test_nn.py

```diff
@@ -10129,6 +10105,34 @@ def test_masked_softmax_TxT_layout(self, device):
         pt_res = self._slow_masked_softmax(input, mask)
         self.assertEqual(pt_res, native_res, exact_dtype=True)
 
+    @onlyCPU
+    @dtypes(torch.bfloat16, torch.half)
+    def test_log_softmax_cpu(self, device, dtype):
```

Comment:
why move it to a different place? It makes reviewing harder. :-)

Reply:
Because it can use `@dtypes` in the original place.